addons/l10n_in_edi/models/res_company.py

```diff
@@ -1,19 +1,38 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-from odoo import fields, models
+from odoo import _, api, fields, models
 
 
 class ResCompany(models.Model):
     _inherit = "res.company"
 
     l10n_in_edi_username = fields.Char("E-invoice (IN) Username", groups="base.group_system")
     l10n_in_edi_password = fields.Char("E-invoice (IN) Password", groups="base.group_system")
-    l10n_in_edi_token = fields.Char("E-invoice (IN) Token", groups="base.group_system")
-    l10n_in_edi_token_validity = fields.Datetime("E-invoice (IN) Valid Until", groups="base.group_system")
+    l10n_in_edi_token = fields.Char("E-invoice (IN) Token",
+                                    groups="base.group_system",
+                                    compute="_compute_edi_token_and_edi_token_validity",
+                                    store=True)
```

Comment:
```suggestion
    l10n_in_edi_token = fields.Char(
        "E-invoice (IN) Token",
        groups="base.group_system",
        compute="_compute_edi_token_and_edi_token_validity",
        store=True
    )
```
@hatr-odoo it should be done like this, it occupies less space indents and gives a good readability apply the same everywhere